### PR TITLE
CTCP-2448: Create new placeholder message for existing movement

### PR DESCRIPTION
**Pull request by liandra-giacomine**

### CODE REVIEW

File: app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala [hunk 1]
```diff
@@ -216,37 +216,59 @@ class MovementsController @Inject() (
...
+      request.headers.get(Constants.ObjectStoreURI).map(ObjectStoreURI.apply) match {
+        case Some(objectStoreURI) => attachLargeMessageInObjectStore(movementId, triggerId, objectStoreURI)
+        case None                 => attachEmptyMessage(movementId)
+      }
+  }
+
+  private def attachEmptyMessage(movementId: MovementId) = {
+    val received = OffsetDateTime.ofInstant(clock.instant, ZoneOffset.UTC)
+
+    val message = messageFactory.createEmptyMessage(
```
Comment (prashanthmamidi) [inline app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala:230]: I think we should status as Pending while creating empty message

Comment (liandra-giacomine) [reply app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala:230]: It does get set to Pending in the messageFactory.createEmptyMessage method doesn't it?

Comment (prashanthmamidi) [reply app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala:230]: yes

**Review by prashanthmamidi — DISMISSED** [review]

**Comment by drnaylor-oc** [comment]
...sorry, I seem to have trashed this PR.

I'll fix it.

File: app/uk/gov/hmrc/transitmovements/repositories/MovementsRepository.scala [hunk 2]
```diff
@@ -307,6 +314,40 @@ class MovementsRepositoryImpl @Inject() (
       case _                  => empty()
     }
 
+  def attachMessage(
```
[the remark this answers is not in the document]
Comment (drnaylor-oc) [reply app/uk/gov/hmrc/transitmovements/repositories/MovementsRepository.scala:317]: `updateMessages`?

[the remark this answers is not in the document]
Comment (liandra-giacomine) [reply app/uk/gov/hmrc/transitmovements/repositories/MovementsRepository.scala:310]: Good spot, removed it now! :)

**Pull request merged**